src/main/webapp/resources/js/components/ontology/OntologyInput.tsx

```diff
@@ -20,7 +28,7 @@ export function OntologyInput({
   onTermSelected,
   ontology,
   autofocus = true,
-}) {
+}: OntologyInputProps): JSX.Element {
   const [options, setOptions] = useState([]);
   const [query, setQuery] = useState("");
```

Comment:
With `useState`  you can type them like : `useState<string>("")`

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f